### PR TITLE
fetching status of keywords

### PR DESCRIPTION
**Pull request by AkhileshNegi**

### CODE REVIEW

File: lib/glific/flows.ex [hunk 1]
```diff
@@ -565,21 +565,26 @@ defmodule Glific.Flows do
...
+              keyword = Glific.string_clean(keyword)
+              Map.put(acc, "published", %{keyword: keyword, id: flow.id})
+            end)
+          if flow.status == "draft", do:
+            Enum.reduce(flow.keywords, acc, fn keyword, acc ->
+              keyword = Glific.string_clean(keyword)
+              Map.put(acc, "draft", %{keyword: keyword, id: flow.id})
+            end)
         end
       )
```
Comment (dlobo) [inline lib/glific/flows.ex:608]: Enum.reduce(flow.keywords, acc, fn keyword, acc ->
               keyword = Glific.string_clean(keyword)
               Map.put(acc, flow.status, %{keyword: keyword, id: flow.id})
             end)
         end

File: lib/glific/flows.ex [hunk 2]
```diff
@@ -565,21 +565,26 @@ defmodule Glific.Flows do
...
 
-    organization = Partners.organization(organization_id)
+      organization = Partners.organization(organization_id)
 
     organization =
```
Comment (dlobo) [inline lib/glific/flows.ex]: Can we also clean up and change the variable name here, it should be called flow_keywords or values or something and not organization

**Comment by pankaj-ag** [comment]
@AkhileshNegi this looks repeated to me.

Should we close this?

**Comment by AkhileshNegi** [comment]
@pankaj-ag  yes

**Pull request closed without merging**